plugins/kotlin-dataframe/testData/testUtils.kt

```diff
@@ -59,3 +62,53 @@ fun compare(runtime: DataFrameSchema, schemas: List<DataFrameSchema>, strict: Bo
     }
 }
 
+// Usual DataFrameSchema.compare is either strict comparison where both set of columns and their type must be the same
+// or subtype relation where subset of columns can vary.
+// This checks that schemas have same set of columns, but compile time columns can be nullable where runtime is narrowed to non-nullable
+
+sealed interface Mismatch
+data class AcceptableMismatch(val path: ColumnPath, val compile: KType, val runtime: KType) : Mismatch
```

Comment:
Probably the nullability fact could be somehow reflected in naming, like NullabilityMismatch or something like this

Reply:
renamed